docs/core/tools/create-custom-templates.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Create custom templates for dotnet new | Microsoft Docs
+description: This topic shows you how to create custom templates for any type of .NET project or files.
+keywords: dotnet new, CLI, CLI command, .NET Core, template, templating
+author: guardrex
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: c28eb59d-4d18-4aba-bcd1-88bd887abc08
+---
+
+# Create custom templates for dotnet new
+
+[!INCLUDE [core-preview-warning](~/includes/core-preview-warning.md)]
+
+The [.NET Core SDK](https://www.microsoft.com/net/download/core) comes with many templates pre-installed for you to use with the [`dotnet new` command](dotnet-new.md), but you can create your own custom templates for any type of .NET project, such as an app, service, tool, or class library. You can even create a template that outputs one or more independent files, such as a configuration file.
+
+Users install your custom template from a NuGet package on any NuGet feed, by referencing a NuGet *nupkg* file directly, or by specifying a file system directory that contains the template. The template engine offers features that allow you to replace values, include and exclude files and regions of files, and execute custom processing operations when your template is used.
+
+The template engine is open source, and the online code repository is at [dotnet/templating](https://github.com/dotnet/templating/) on GitHub. Visit the [dotnet/dotnet-template-samples](https://github.com/dotnet/dotnet-template-samples) repo for samples of templates. More templates, including templates from third parties, are found at [Available templates for dotnet new](https://github.com/dotnet/templating/wiki/Available-templates-for-dotnet-new) on GitHub. A detailed pre-release blog post that describes how to create and install custom templates is at [How to create your own templates for dotnet new](https://blogs.msdn.microsoft.com/dotnet/2017/04/02/how-to-create-your-own-templates-for-dotnet-new/).
```

Comment:
> A detailed pre-release blog post that describes how to create and install custom templates is at How to create your own templates for dotnet new.

What makes the blog post useful to me if I'm already reading this docs article about how to create templates? Maybe you should mention why I should read the blog post (instead of continuing reading this article), or remove the link altogether?